src/Service/TraceContext/TraceContextService.php

```diff
@@ -43,10 +41,22 @@ public function createNewTrace(): TraceContext
         return $trace;
     }
 
+    public function createTraceFrom(string $traceId): TraceContext
```

Comment:
Naming; what you're passing is not Just the traceId. But the full version-traceid-transactionid-flag string

Reply:
Renamed it to `$traceParent` similar to the variable in called methods.